pkg/interop/util/util.go

```diff
@@ -22,6 +26,34 @@ func FromAddress(address string) interop.Hash160 {
 	return nil
 }
 
+// AddressToHash160 is a utility function that converts a Neo address to its hash
+// (160 bit BE value in a 20 byte slice). Unlike the FromAddress, AddressToHash160
+// will generate the honest Neo-VM script for runtime address conversion, thus
+// AddressToHash160 may be used for strings got at runtime. For strings those value
+// is known at compilation time, use FromAddress for the sake of lower script cost.
+func AddressToHash160(address string) interop.Hash160 {
```

Comment:
How about making a `pkg/interop/neogo/address` for this? I'd like to separate our library functions (we can have a number of them) from core functionality. `util` kinda does that, but `util` is always a bad package name. This way we could have `address.FromHash160` and `address.ToHash160` (or mimic `encoding/address`).

Reply:
I like the idea and also thought about it, fixed. But maybe we can replace `neo-go` with something more appropriate? Otherwise import path looks like `"github.com/nspcc-dev/neo-go/pkg/interop/neo-go/address"`.

Reply:
It should not have a dash, I think. But other than that I don't have a lot of ideas. `lib`? `golib` `libgo`?

Reply:
Maybe `lib` will go? I've implemented it.